jira/client.py

```diff
@@ -4381,6 +4381,26 @@ def delete_permissionscheme(self, id: str):
         self.permissionschemes.cache_clear()
         return data
 
+    def get_form(self, issue: str, formid: str) -> list[dict]:
+        """For the specified issue, returns the form data as key/value list. (Admin required).
+
+        Args:
+            issue (str): The issue id.
+            formid (str): the ID of the form, default is 1
+
+        Returns:
+            List[dict]: A dictionary containing the "answers" as value and the label as key.
+        """
+        response = self._session.get(
```

Comment:
have a look at thee function below and try and copy the style.
We want to use the `_get_url` function with f-strings based on the user input